frontend/coverage/lcov-report/src/stores/cost-store.ts.html

```
@@ -0,0 +1,352 @@
+                <span class='fraction'>6/27</span>
+            </div>
+        
+            
+        </div>
+        <p class="quiet">
+            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
+        </p>
+        <template id="filterTemplate">
+            <div class="quiet">
+                Filter:
+                <input type="search" id="fileSearch">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)